templates/rhpam74-prod-immutable-kieserver-amq.yaml

```diff
@@ -688,6 +803,8 @@ objects:
     to:
       kind: "Service"
       name: "${APPLICATION_NAME}-amq-jolokia"
+    tls:
```

Comment:
does the port  supports tls without configuration?

Reply:
It's added to allow access to the AMQ console. I wasn't able to access the console when the secret was added to the AMQ